www/app/api/chat/honcho/route.ts

```diff
@@ -24,5 +24,7 @@ export async function POST(req: NextRequest) {
     { queries: message }
   );
```

Comment:
The query to the Dialectic Endpoint shouldn't be just the user input message but should also include the query the thought chain came up with in the <honcho> tags